plugin.js

```diff
@@ -42,6 +63,15 @@ Plugin.prototype.exec = function (server, cb) {
 
   debug('exec', name)
 
+  var timer
+
+  if (this.parent._timeout > 0) {
+    timer = setTimeout(function () {
+      timer = null
+      done(new Error('plugin did not start in time: ' + name))
```

Comment:
I would rephrase this as `plugin '${name}' timed out.`

Reply:
I didn't do that because name can be quite a long string (the first two lines of the function in the bad case). Having those in the middle would make the error unreadable.